src/js/login/login-entry.jsx

```diff
@@ -1,16 +0,0 @@
-import '../../sass/login.scss';
-
-import React from 'react';
-import startReactApp from '../../platform/startup/react';
-
-import { Provider } from 'react-redux';
-
-import Main from './containers/Main';
-
-export default function createLoginWidget(store) {
-  startReactApp((
-    <Provider store={store}>
-      <Main/>
-    </Provider>
-  ), document.getElementById('login-root'));
-}
```

Comment:
Renamed the module to `index.js` as a top level module that exports the initializer for the app. `feedback-entry` and `login-entry` were kind of misleading in that these are usually the names of entry points for metalsmith pages, and they didn't correspond to any.